docs/src/solvers/ode_solve.md

```diff
@@ -180,6 +180,11 @@ Zhang and Shu (Zhang, Xiangxiong, and Chi-Wang Shu. "Maximum-principle-satisfyin
 high-order schemes for conservation laws: survey and new developments." Proceedings of the Royal Society of
 London A: Mathematical, Physical and Engineering Sciences. The Royal Society, 2011.).
 
+### Low-Storage Methods for Hyperbolic PDEs (Conservation Laws)
```

Comment:
Are these specifically for Hyperbolic PDEs? I thought low storage methods are just in general for large systems.

Reply:
Yes, they can be used in in general for large system, but they have been designed especially for hyperbolic equations (since the authors are interested in conservation laws).

Reply:
What about these are specific to hyperbolic equations?